frontend/src/cards/UnknownsMapCard.tsx

```diff
@@ -222,7 +227,16 @@ function UnknownsMapCardWithKey(props: UnknownsMapCardProps) {
                 <ChoroplethMap
                   activeDemographicGroup={UNKNOWN}
                   countColsMap={{}}
-                  data={unknowns as DataPoint[]}
+                  data={
+                    unknowns.map((unknown) => {
+                      const value = unknown[metricConfig.metricId]
+                      return {
+                        ...unknown,
+                        [metricConfig.metricId]:
+                          value == null ? undefined : value,
+                      }
+                    }) as DataPoint[]
+                  }
```

Comment:
I don't think this is needed, since you're specifically checking for either `undefined` or `null` in tooltipUtils; try just using `data={unknowns}` here and see if everything is working as expected